### PR TITLE
(fix): Updating mapping for associations & fixing AP ANPA dateline parsing

### PR DESCRIPTION
**Pull request by mdhaman**
- To use filtering based on featuremedia.
- Fixed the failing AP ANPA dateline parsing.

### CODE REVIEW

File: superdesk/io/feed_parsers/ap_anpa.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def ap_derive_dateline(self, item):
             html = item.get('body_html')
             if html:
                 parsed = parse_html(html, content='html')
-                for par in parsed.xpath('/html/div/child::*'):
+                for par in parsed.xpath('/html/body/child::*'):
```
Comment (naspeh) [inline superdesk/io/feed_parsers/ap_anpa.py]: is this change occasional?

Comment (naspeh) [inline superdesk/io/feed_parsers/ap_anpa.py]: yep, I see, just misunderstood with subject (which says only about mapping)

Comment (mdhaman) [inline superdesk/io/feed_parsers/ap_anpa.py]: PR was about mapping only but unit test failed. 
In version 1.7 xpath needs to `/html/body` but in version 1.8 it needs to be `/html/div`. It looks like changes were done after `root element` changes in version 1.8. (https://github.com/superdesk/superdesk-core/blob/master/superdesk/etree.py#L121)

Comment (naspeh) [inline superdesk/io/feed_parsers/ap_anpa.py]: thanks for clarification, now I certainly understand this.

**Pull request merged**